File: app/src/main/java/github/hellocsl/cursorwheellayout/data/ImageData.java
package github.hellocsl.cursorwheellayout.data;

public class ImageData {
    public int mImageRes;
    public String mImageRes_1;
    public String mDesc;

    public ImageData(int imageRes, String desc) {
        mImageRes = imageRes;
        mDesc = desc;
    }

    public ImageData(String imageRes, String desc) {
        mImageRes_1 = imageRes;
        mDesc = desc;
    }
    public int getIMageRes(){
        return mImageRes;
    }

    public String getmImageRes_1(){
        return mImageRes_1;
    }
    public String getmDesc(){
        return mDesc;
    }
}

File: app/src/main/java/github/hellocsl/cursorwheellayout/data/ImageData.kt
package github.hellocsl.cursorwheellayout.data

class ImageData {
    var iMageRes: Int = 0
    lateinit var mImageRes_1: String
    var mDesc: String

    constructor(imageRes: Int, desc: String) {
        iMageRes = imageRes
        mDesc = desc
    }

    constructor(imageRes: String, desc: String) {
        mImageRes_1 = imageRes
        mDesc = desc
    }

    fun getmImageRes_1(): String {
        return mImageRes_1
    }

    fun getmDesc(): String {
        return mDesc
    }
}
